derek/ner/nn/classifier.py

```diff
@@ -33,15 +33,20 @@ def __init__(self, graph, feature_extractor: NERFeatureExtractor, feature_comput
         self.saver = saver
         self.post_processor = post_processor
 
-    def predict_doc(self, doc: Document) -> List[Entity]:
+    def predict_doc_with_scores(self, doc: Document) -> Tuple[List[Entity], List[float]]:
+        """
+        :param doc: document
+        :return: (list of labels sequences for each sentence in document, score for each sequence)
```

Comment:
no, it returns `(List[Entity] found in doc, List[float] containing sequence labelling score for each Sentence in doc)`